src/tests/simulation/src/main.rs

```diff
@@ -156,7 +174,7 @@ async fn main() {
                 ]);
                 risingwave_compute::start(opts).await
             });
-        if args.kill_node {
+        if args.kill_compute || args.kill_meta {
```

Comment:
Should the compactor also restart on meta restarts? 🤔

Reply:
Not tested yet. I will add this to compactor if needed.